socket.yml

```diff
@@ -67,6 +67,15 @@ endpoints:
         customerId:
           type: string
           description: Unique user id
+  migrate:
+    description: username to customerId migration
+    required:
+      - migrationKey
+    inputs:
+      properties:
+        migrationKey:
+          type: string
```

Comment:
`required` property doesn't check if value is not empty, it only checks if key is present in request data. Add `minLength` for extra validation